packages/components/src/KaizenProvider/KaizenProvider.tsx

```diff
@@ -27,13 +28,13 @@ export const KaizenProvider = ({
 
   return (
     <OptionalIntlProvider locale={locale}>
-      <>
+      <I18nProvider locale={locale}>
```

Comment:
This is a bit confusing, maybe rename it to something specfic to React Aria so that we don't mix this up with `@cultureamp/i18n-react-intl` 

And consider putting it together in the `OptionalIntlProvider` but not make it optional